pkgs/applications/misc/urlscan/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildPythonPackage, isPy35, fetchFromGitHub, urwid }:
+
+buildPythonPackage rec {
+  name = "urlscan-${version}";
+  version = "0.8.3";
+
+  src = fetchFromGitHub {
+    owner = "firecat53";
+    repo = "urlscan";
+    rev = version;
+    # (equivalent but less nice(?): rev = "00333f6d03bf3151c9884ec778715fc605f58cc5")
+    sha256 = "0l40anfznam4d3q0q0jp2wwfrvfypz9ppbpjyzjdrhb3r2nizb0y";
+  };
+
+  propagatedBuildInputs = [ urwid ];
+
+  # FIXME doesn't work with 2.7; others than 2.7 and 3.5 were not tested (yet)
+  disabled = ! isPy35;
```

Comment:
Thank you!  However, shouldn't it be `!pythonOlder "3.5"` then (since `pythonOlder` returns false for the same version)?